README.md

```diff
@@ -115,17 +117,37 @@ foundryup
 curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
 ```
 
+### Installing Colima
+
+```bash
+brew install colima
+colima start
+```
+
+### Installing Docker
+
+```bash
+brew install docker
+```
+
 ## Setup and Execution
 
+You can run the AVS both locally using `anvil` or on Holesky testnet.
+
 A `Makefile` is provided to ease the execution. Follow these steps in separate terminals:
 
-1. Build contracts: `make build-contracts`
-2. Start local blockchain: `make anvil`
-3. Deploy contracts: `make deploy-contracts`
-4. Launch first Operator: `make run-operator-uji`
-5. Launch second Operator: `make run-operator-floki`
-6. Start Aggregator: `make run-aggregator`
-7. Create a test task: `make create-task`
+### Local Execution
+
+1. Start local blockchain: `make anvil`
```

Comment:
BTW, you can use `1.` for all list items, the renderer will take care of putting the real numbers

```suggestion
1. Start local blockchain: `make anvil`
1. Deploy SC: `make deploy-contracts-anvil`
```

Reply:
We no longer need to deploy contracts since they are deployed on testnet when we ```make anvil``` it fork the current state with the deployed contracts

Reply:
[1. for list](https://github.com/gizatechxyz/avsthon/pull/11/commits/4cc93f3dcc6a84d0d9496ea732c20c78cbfa59b4)